ban/core/resource.py

```diff
@@ -41,9 +42,10 @@ def _validate_coerce(self, coerce, field, value):
     def validate(self, data, instance=None, **kwargs):
         self.instance = instance
         super().validate(data, **kwargs)
-        if ('version' in self.schema and instance
-           and not self.document.get('version')):
-            self._error('version', errors.ERROR_REQUIRED_FIELD)
+        if hasattr(self.model, 'validate'):
+            for key, message in self.model.validate(self, self.document,
+                                                    instance).items():
+                self._error(key, message)
```

Comment:
You might want [to use a different signature](https://github.com/nicolaiarocci/cerberus/blob/master/cerberus/cerberus.py#L177):

```self._error(*[errors in self.model.validate(self, self.document, instance).items()])``` (pseudo-code)